examples/multilabel_classification/1-quickstart/plot_tutorial_multilabel_classification.py

```diff
@@ -208,3 +213,73 @@
         fontsize=20
     )
 plt.show()
+
+##############################################################################
+# 3. Precision control risk with LTT
+# ------------------
+# 3.1 Fitting MapieMultilabelClassifier
+# -------------------------------------
+#
+# In this part, we will use LTT to control precision.
+# At the opposite of the 2 previous method, LTT can handle non-monotonous loss.
+# The procedure consist in multiple hypothesis testing. This is why the output
+# of this procedure isn't reduce to one value of :math:`\lambda`.
+#
+# More precisely, we look after all the :math:`\lambda` that sastisfy the
+# following:
+# :math:`\mathbb{P}(R(\mathcal{T}_{\lambda}) \leq \alpha ) \geq 1 - \delta`,
+# where :math:`R(\mathcal{T}_{\lambda})` is the risk we want to control and
+# each :math:`\lambda`` should satisfy FWER control.
+# :math:`\alpha` is the desired risk.
+#
+# Notice that the procedure will diligently examine each :math:`\lambda`
+# such that the risk remains below level :math:`\alpha`, meaning not
+# every :math:`\lambda` will be considered.
+# This means that a for a :math:`\lambda` such that risk is below
+# :math:`\alpha`
+# doesn't necessarly pass the FWER control! This is what we are going to
+# explore.
+
+mapie_clf = MapieMultiLabelClassifier(
+    estimator=clf,
+    method='ltt',
+    metric_control='precision'
+)
+mapie_clf.fit(X_cal, y_cal)
+
+alpha = 0.1
+_, y_ps = mapie_clf.predict(
+    X_test,
+    alpha=alpha,
+    delta=0.1
+)
+
+valid_index = mapie_clf.valid_index[0]  # valid_index is a list of list
+
+lambdas = mapie_clf.lambdas[valid_index]
+
+mini = lambdas[np.argmin(lambdas)]
+maxi = lambdas[np.argmax(lambdas)]
+
+r_hat = mapie_clf.r_hat
+idx_max = np.argmin(r_hat[valid_index])
+
+##############################################################################
+# 3.2 Results
+# -----------
+# We can see that not all :math:`\lambda` such that risk is below the orange
+# line are choosen by the procedure. Otherwise, all the lambdas that are
+# in the red rectangle verify fwer control and allow to control precision
```

Comment:
*fewer

Reply:
```suggestion
# in the red rectangle verify fewer control and allow to control precision
```

Reply:
It is actually fwer for family wise error rate. I will change it to family wise error rate to be more precise